solution/ui/regulations/eregs-component-lib/src/components/shared-components/Statutes/schemas/tableSchemas.js

```diff
@@ -1,4 +1,10 @@
-import { houseGovUrl, statuteCompilationUrl, usCodeUrl, ssaGovUrl } from "../utils/urlMethods.js";
+import {
+    houseGovUrl,
+    statuteCompilationUrl,
+    usCodeUrl,
```

Comment:
Doesnt need to be done here but should we maybe have our imports sorted alphabetically. Its currently what we do with Django.  Granted we have Ruff automatically order them for us so its much easier to manage.

Reply:
alphabetized!